src/content/components/BugList/columnTransforms.js

```diff
@@ -145,9 +145,17 @@ export const columnTransforms = {
   last_change_time(value) {
     return formatDate(value, true);
   },
-  priority(priority) {
+  priority(priority, _, props) {
+    function openPriorityGuide() {
+      props.priorityGuide.current.openPriorityGuide();
+    }
+
     return (
-      <span className={priorityStyles[priority.toLowerCase()]}>{priority}</span>
+      <button
+        className={priorityStyles[priority.toLowerCase()]}
+        onClick={priority == "--" ? null : openPriorityGuide.bind(this)}>
```

Comment:
Are you sure we need to pass so many props around? This is the kind of thing I added the GlobalContext for. It might make sense to make a component called PrefObserver that stores pref values, gets them from localStorage at startup, mirrors them in localStorage, and allows stuff to subscribe to changes. This is kinda hard to explain so let me try to draft something like that and then you can see if it works for this

Reply:
See my PR #259

Reply:
I don't think this will work since localStorage stores data in strings. I need to store a React Ref of PriorityGuide so that I can call one of its functions from another component.

Reply:
It works fine. I already tested it last week. The prefs storage stringifies and parses values automatically. Wasn't it you who added that feature?

You just need to add a subscription callback in the PriorityGuide constructor or on mounted. Just like I did for the other components in #259.

```js
prefs.on("priorityGuideCollapsed", this.onPrefChange);
...
  onPrefChange({ name, newValue } = {}) {
    if (name === "priorityGuideCollapsed") {
      this.setState({ priorityGuideCollapsed: newValue });
    }
  }
```

Then your buttons don't need to do anything except
```js
prefs.set("priorityGuideCollapsed", false);
```

Reply:
Btw, the pref name should be `priority_guide_collapsed` or something. Since all the other pref names are already snake cased, not camel cased

Reply:
Thanks! I was misunderstanding something about your suggestion previously